cli/run_util.go

```diff
@@ -438,8 +439,13 @@ func (r WorkflowRunner) executeStep(
 
 func (r WorkflowRunner) runStep(
 	stepUUID string,
-	step stepmanModels.StepModel, stepIDData models.StepIDData, stepDir string,
-	environments []envmanModels.EnvironmentItemModel, secrets []string) (int, []envmanModels.EnvironmentItemModel, error) {
+	step stepmanModels.StepModel,
+	stepIDData models.StepIDData,
+	officialStep bool,
```

Comment:
Is officialStep used?

Reply:
Nice catch. I forgot to remove it but now it is done.